src/Microsoft.Dnx.Host.CoreClr/DomainManager.cs

```diff
@@ -11,8 +12,24 @@
 [SecurityCritical]
 sealed class DomainManager
 {
+    [StructLayout(LayoutKind.Sequential)]
+    private unsafe struct NativeBootstrapperContext
+    {
+        public char* OperatingSystem;
+
+        public char* OsVersion;
+
+        public char* Architecture;
+
+        public uint NumProcessors;
```

Comment:
@DamianEdwards asked for that - they wanted to use this in the perf tests I believe.